service/http/dffml_service_http/routes.py

```diff
@@ -342,6 +359,30 @@ async def multicomm_dataflow(self, config, request):
                         status=HTTPStatus.NOT_FOUND,
                     )
 
+    async def multicomm_dataflow(self, config, request):
+        if config.immediate_response:
+            asyncio.create_task(self._multicomm_dataflow(config, request))
+
+            ir = config.immediate_response
+            content_type = ir["content_type"]
```

Comment:
We should auto determine the content type here if it's not given. For example

- If it doesn't exist and
  - data exists
    - application/json
  - text exists
    - text/plain
  - body exists
    - application/octet-stream